org.eclipse.scout.rt.dataobject/src/main/java/org/eclipse/scout/rt/dataobject/id/IdInventory.java

```diff
@@ -46,6 +51,22 @@ protected void createClassCache() {
       }
     }
     LOG.debug("Registered {} id types", m_nameToClassMap.size());
+
+    ClassInventory.get().getKnownAnnotatedTypes(IdEncryption.class).stream()
```

Comment:
I love using streams - but if the code written with streams is longer and requires a lot of javadoc in order to make it readable for others I prefer using classic for-loops (code is more compact, readable and matches the existing pattern in this class)
```
    for (IClassInfo classInfo : ClassInventory.get().getKnownAnnotatedTypes(IdEncryption.class)) {
      Class<?> idClass = classInfo.resolveClass();
      if (idClass.isAssignableFrom(IId.class)) {
        //noinspection unchecked
        registerIdEncryption((Class<? extends IId>) idClass, idClass.getAnnotation(IdEncryption.class).value());
      } else {
        LOG.warn("Class {} is annotated with @{} but does not implement {}. Skipping class.", idClass.getName(), IdEncryption.class.getSimpleName(), IId.class.getName());        
      }
    }
    LOG.debug("Registered {} id encryption mappings", m_classToIdEncryptionMap.size());
```